docs/reference/api/field-formats.md

```diff
@@ -0,0 +1,245 @@
+# API response field examples
+
+Simple examples of how each field type might be returned in a document from the REST APIs.
+
+## area
+
+```json
+// An area with a single rich text widget
+{
+  "main": {
+    "_id": "ckj0k0dy7000i2a68s1z8v4ky",
+    "items": [
+      {
+        "_id": "ckj0k0mox000o2a68oouznar1",
+        "metaType": "widget",
+        "type": "@apostrophecms/rich-text",
+        "content": "<p>Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum <strong>nibh, ut fermentum massa justo sit amet risus.</strong></p><p><br /></p>",
+        "_docId": "ckj0k2i45001c7u9kky3tftx2"
+      }
+    ],
+    "metaType": "area",
+    "_docId": "ckj0k2i45001c7u9kky3tftx2"
+  }
+}
+```
+## array
+
+```json
```

Comment:
Should be `javascript` json does not allow comments and this breaks the formatting

Reply:
In this case the responses are JSON, so I'd like to keep using that. It's simply a signal for which code highlighter to use and the comments do look like comments in the end result.

Reply:
I don't think this is a good idea. It's not there for programmers to read (they can't see it when they read the docs), it's there for the markdown renderer to read, and it's the wrong signal even if it happens to be working right now.

Reply:
Also confusing when reading the PR (the comments are flagged as errors).